composer/modules/js-tests/src/test/resources/failing/sql-connector.bal

```diff
@@ -40,7 +40,7 @@ function main (string[] args) {
     println("Inserted row count:" + count);
     println("Generated key:" + ids[0]);
 
-    //Select data using select action. Select action returns a datatable
+    //SelectElements data using select action. SelectElements action returns a datatable
```

Comment:
Did you use s/Select/SelectElements/g? This change should be reverted.

Reply:
@hasithaa I fixed this in recent commits